### ISSUE TITLE
Questions about experiment

### ISSUE THREAD

**Issue by LiuQiir** [supersyq/RIENet#6]
Thank you for your work, and I hope to learn some knowledge from it. When using your pretrained model for testing, there is a significant discrepancy in the MIE(R) value compared to the data provided in the paper. This issue occurs for all datasets except KITTI. Could you please explain the reason for this?
![1d9cd68aede1e8e994087975155a5c1](https://github.com/supersyq/RIENet/assets/116939982/d4bb51dc-ef25-4f23-bf41-ec37dad0c20a)

**Comment by supersyq** [comment]
I am sorry for my delay. We have tested the pre-trained models on all datasets. The results are similar to those listed in our paper. And there is no significant discrepancy in the MIE(R). Here are the results: (7scenes->ICL->same->noise->unseen>)

![Uploading 7scenes.PNG…]()

**Comment by supersyq** [comment]
I am sorry for my delay. We have tested the pre-trained models on all datasets. The results are similar to those listed in our paper. And there is no significant discrepancy in the MIE(R). Here are the results(7scenes->ICL->same->noise->unseen):
![7scenes](https://github.com/supersyq/RIENet/assets/96189873/9059b417-9e6e-498f-a692-0fbb613a0e29)

**Comment by LiuQiir** [comment]
> I am sorry for my delay. We have tested the pre-trained models on all datasets. The results are similar to those listed in our paper. And there is no significant discrepancy in the MIE(R). Here are the results(7scenes->ICL->same->noise->unseen): ![7scenes](https://private-user-images.githubusercontent.com/96189873/291799140-9059b417-9e6e-498f-a692-0fbb613a0e29.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUwNjg4NDMsIm5iZiI6MTcwNTA2ODU0MywicGF0aCI6Ii85NjE4OTg3My8yOTE3OTkxNDAtOTA1OWI0MTctOWU2ZS00OThmLWE2OTItMGZiYjYxM2EwZTI5LlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTEyVDE0MDkwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUyZjBmN2Q3ODJhNTc0MmFlOWU2MjJiM2Q2ZDY3YWQxYmQ0M2UyOTY5ZjQ0OWM4YWQ0NmJhMzhmNjA1NWVkNDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.A7QGfb9bMu1rrzHsRYML0wkuL_b-Bk-m3LxLEZWnj8g)

Thank you for your reply, I apologize that I just saw it. I still have a question, as your figures shows, For datasets other than kitti, the metric MIE(R) is much larger than expected. I hope you can help me to answer it!
